src/components/field-array-source/field-array-source.component.tsx

```diff
@@ -31,46 +38,47 @@ export const FieldArraySource = (props): JSX.Element => {
         <div className="col-sm-7" />
       </div>
 
-      {_.get(props, 'form.values.kildebeskrivelse.forholdTilKilde') != 'egendefinert' && (
-        <div>
-          {_.get(props, 'form.values.kildebeskrivelse.kilde', []).map((kilde, index) => {
-            return (
-              <div key={kilde.id} className="row d-flex fdk-after-element">
-                <div className="col-sm-5">
-                  <Field
-                    name={`kildebeskrivelse.kilde[${index}].tekst`}
-                    component={InputField}
-                    label={localization.titleSource}
-                    showLabel={true}
-                  />
-                </div>
+      {!(_.get(props, 'form.values.kildebeskrivelse.forholdTilKilde') == 'egendefinert') &&
```

Comment:
for nullsafe så kan du si  _.get(props, [form, 'values', 'kildebeskrivelse', 'forholdTilKilde']